packages/fxa-settings/src/components/PageTwoStepAuthentication/index.tsx

```diff
@@ -197,7 +226,7 @@ export const PageTwoStepAuthentication = (_: RouteComponentProps) => {
 
   return (
     <FlowContainer
-      title="Two Step Authentication"
+      title={l10n.getString('tfa-title')}
```

Comment:
Oh this makes more sense than what I was trying to do, passing in the title as a attr property.